blueprint.json

```
@@ -0,0 +1,54 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+    "landingPage": "/wp-admin/admin.php?page=tainacan_admin#/home",
```

```suggestion
    "landingPage": "/wp-admin/admin.php?page=tainacan_admin",
```
Vamos deixar este link mais genérico. Assim se no futuro mudarmos o link de home ele cai no default.